src/Pages/ReferenceTokens.razor

```diff
@@ -87,6 +88,11 @@
     
     private void CheckReferenceToken(EditContext editContext)
     {
+        var logged = databaseAuditLog.Log("ReferenceTokenChecked");
+        if (logged != 1)
```

Comment:
I would argue that the exception handling of not being able to write to the audit log is the responsibility of the implementation of the interface, rather than doing it here in the component. Perhaps writing an exception to an ILogger<T> could be enough as an exception handling mechanism.

Reply:
changed it in 27976c1